lib/zendesk-uploader/client-wrapper.js

```diff
@@ -68,7 +72,11 @@ module.exports = class ZendeskClientWrapper {
                     // err.retryAfter is present on all 429 (rate limit has been exceeded) and some 503 errors
                     // err.retryAfter is time to wait in seconds
                     retries--;
-                    let retryAfter = err.retryAfter ? err.retryAfter * 1000 : RETRY_AFTER_DEFAULT;
+                    retryNumber++;
+                    const retryAfter = err.retryAfter
+                        ? err.retryAfter * 1000
+                        // Default retry should wait twice as long as the previous retry
+                        : RETRY_AFTER_DEFAULT * Math.max(Math.pow(2, retryNumber - 1), 1);
```

Comment:
I've added a `MAX_RETRY_TIME` set to 8000ms. I guess this setting could go in the config as well.